library/blivet.py

```diff
@@ -700,6 +702,11 @@ def action_dict(action):
             module.fail_json(msg=str(e), **result)
 
     scheduled = b.devicetree.actions.find()
+    result['packages'] = b.packages[:]
+
+    if module.params['packages_only']:
+        module.exit_json(**result)
+
```

Comment:
This won't work because if the packages are unavailable, we have already failed and exited after the manage_pool call (which tracebacks) a few lines above.